db/primary_migrate/20231024172229_load_email_data.rb

```diff
@@ -0,0 +1,9 @@
+class LoadEmailData < ActiveRecord::Migration[7.0]
+  def change
+    ActiveRecord::Base.connection.execute <<~PG
+      COPY disposable_domains (name)
+      FROM '#{EmailData.data_dir.join('disposable_domains')}.txt'
+      (FORMAT CSV)
```

Comment:
+1 to @mitchellhenke's suggestion about loading incrementially

Another thought is, what about duplicates? Should we add a unique index and add `ON DUPLICATE KEY IGNORE` (or its equivalent in psql)?

Reply:
I added a unique index to the create query, but should be a good idea to add the on duplicate key ignore